src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -25,24 +25,36 @@ EOF
 }
 
 function clone () {
-    curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+    if [ $TEST_TYPE == "local" ]; then
+        curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+    elif [ $TEST_TYPE == "kube" ]; then
+        kubectl exec -i $CODEWIND_POD_ID -- curl -H "Content-type: application/json" -d '{ "projectName": "'$1'" , "parentPath": "'$2'", "url": "'$3'" }' "https://localhost:9191/api/v1/projects" --insecure
+    fi
 }
 
 function setup {
     DATE_NOW=$(date +"%d-%m-%Y")
     TIME_NOW=$(date +"%H.%M.%S")
-    BUCKET_NAME=turbine-$TEST_SUITE
-    TEST_DIR=$CW_DIR/src/pfe/file-watcher/server
-    TEST_DIR_CONTAINER=/file-watcher/server
-    TEST_OUTPUT_DIR=~/test_results/$TEST_SUITE/$DATE_NOW/$TIME_NOW
+    BUCKET_NAME=turbine-$TEST_TYPE-$TEST_SUITE
+    TURBINE_SERVER_DIR=$CW_DIR/src/pfe/file-watcher/server
+    TEST_DIR=$TURBINE_SERVER_DIR/test
+    TURBINE_DIR_CONTAINER=/file-watcher
+    TEST_OUTPUT_DIR=~/test_results/$TEST_TYPE/$TEST_SUITE/$DATE_NOW/$TIME_NOW
     TEST_OUTPUT=$TEST_OUTPUT_DIR/test_output.xml
-    TEST_LOG=$TEST_OUTPUT_DIR/$TEST_SUITE-test.log
-    CODEWIND_CONTAINER_ID=$(docker ps | grep codewind-pfe-amd64 | cut -d " " -f 1)
     PROJECTS_CLONE_DATA_FILE="./resources/projects-clone-data"
-
-    mkdir -p $TEST_OUTPUT_DIR \
-    && docker exec -it $CODEWIND_CONTAINER_ID bash -c "cd /file-watcher/server; npm install --only=dev" \
-    && docker cp $TEST_DIR/test $CODEWIND_CONTAINER_ID:$TEST_DIR_CONTAINER
+    TEST_LOG=$TEST_OUTPUT_DIR/$TEST_TYPE-$TEST_SUITE-test.log
+
+    mkdir -p $TEST_OUTPUT_DIR
+
+    if [ $TEST_TYPE == "local" ]; then
+        CODEWIND_CONTAINER_ID=$(docker ps | grep codewind-pfe-amd64 | cut -d " " -f 1)
+        docker cp $TURBINE_SERVER_DIR $CODEWIND_CONTAINER_ID:$TURBINE_DIR_CONTAINER \
+        && docker exec -it $CODEWIND_CONTAINER_ID bash -c "cd /file-watcher/server; npm install --only=dev"
+    elif [ $TEST_TYPE == "kube" ]; then
+        CODEWIND_POD_ID=$(kubectl get po --selector=app=codewind-pfe --show-labels | tail -n 1 | cut -d " " -f 1)
+        kubectl cp $TURBINE_SERVER_DIR $CODEWIND_POD_ID:$TURBINE_DIR_CONTAINER \
+        && kubectl exec -it $CODEWIND_POD_ID -- bash -c "cd /file-watcher/server; npm install --only=dev"
```

Comment:
Can we make the command `"cd /file-watcher/server; npm install --only=dev"` a variable and call it for both docker and kube exec? So in the future if it changes we only need to update it in one place

Reply:
tried it, it did not work

Reply:
What was the error?